framework/components/ecl-buttons/_buttons.scss

```diff
@@ -139,6 +139,12 @@
   }
 }
 
+.ecl-button--splash-page--is-active {
```

Comment:
A BEM selector can't have 2 modifiers. I think in this case we can accept `.ecl-button--splash-page.ecl-button--active`